main/main.cpp

```diff
@@ -501,6 +502,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 		} else if (I->get() == "--version") {
 			print_line(get_full_version_string());
+			print_line("Build arguments: " + String(BUILD_ARGUMENTS));
```

Comment:
I don't know if putting it in `--version` is the best, as some might rely on parsing it to figure out the Godot version, and the kind of output this would add can be an annoyance. This should maybe go to a new command line option (possibly repeating the version string too).

Reply:
Maybe `--build-args`?